frontend/src/stores/user.js

```diff
@@ -1,5 +1,6 @@
 import { defineStore } from "pinia";
 import { get_request } from "./services/request_http";
+import { useAuthStore } from "./auth"; // Asegúrate de importar el store de auth
```

Comment:
Change comment to english

Reply:
Addressed in [commit](https://github.com/carlos18bp/gym_project/pull/5/commits/75f045424c57e6fbc9fb053661f5d23294a22f09)